package.json

```diff
@@ -48,7 +47,104 @@
       "react/sort-comp": 0,
       "jest/valid-expect": 0,
       "jest/no-disabled-tests": 0,
-      "react/no-find-dom-node": 0
+      "react/no-find-dom-node": 0,
```

Comment:
please extract this

also see https://github.com/developit/preact-cli/blob/next/.eslintrc#L4